systemdspawner/systemdspawner.py

```diff
@@ -70,6 +69,21 @@ class SystemdSpawner(Spawner):
         """
     ).tag(config=True)
 
+    users_without_quotas = Set(
```

Comment:
Hi and thanks for the PR :-)

Perhaps this would be better as a Bool similar to https://github.com/jupyterhub/jupyterhub/blob/master/jupyterhub/app.py#L453

Reply:
So, in that case, we would enable this 'no quotas mode' for all user in the admin group ? So you want to check that the user is in `Authenticator.admin_users` and that  `users_without_quotas` is `True` ?

Reply:
Sorry @RobinFrcd. I didn't mean to confuse. The overarching point that I'm trying to make (though not eloquently) is that any time a user (admin or otherwise) has the ability to use all the resources that should be an opt-in action instead of the default.

Reply:
Well, if a user is in `users_without_quotas` he has the ability to use all the resources. If he's not in, he can't. So the default is that a user **can't** use all the resources. I think I don't get the point here ! :)